bokehjs/src/lib/models/annotations/size_bar.ts

```diff
@@ -0,0 +1,345 @@
+      // ...mixins.attrs_of(this.model, "border_", mixins.Line, true),
+      ...mixins.attrs_of(this.model, "bar_", mixins.Line, "outline_"),
+    }
+
+    switch (orientation) {
+      case "horizontal": {
+        this._size_bar = new InternalPlot({
+          width_policy: width == "max" ? "max" : "fit",
+          height_policy: height == "max" ? "max" : "fit",
+          frame_width: width == "max" ? undefined : width,
+          frame_height: height == "max" ? undefined : height,
+          below: [this._major_axis],
+          x_range: this._major_range,
+          y_range: this._minor_range,
+          x_scale: this._major_scale,
+          y_scale: this._minor_scale,
+          ...plot_attrs,
+        })
+        break
+      }
+      case "vertical": {
+        this._size_bar = new InternalPlot({
+          width_policy: height == "max" ? "max" : "fit",
+          height_policy: width == "max" ? "max" : "fit",
+          frame_width: height == "max" ? undefined : height,
+          frame_height: width == "max" ? undefined : width,
+          right: [this._major_axis],
+          x_range: this._minor_range,
+          y_range: this._major_range,
+          x_scale: this._minor_scale,
+          y_scale: this._major_scale,
+          ...plot_attrs,
+        })
+        break
+      }
+    }
+  }
+
+  override get computed_elements(): ElementLike[] {
+    return [...super.computed_elements, this._size_bar]
+  }
+
+  override async lazy_initialize(): Promise<void> {
+    await super.lazy_initialize()
+    this._size_bar_view = this._element_views.get(this._size_bar) as ViewOf<Plot>
+  }
+
+  override update_layout(): void {
+    this.layout = this._size_bar_view.layout
+    this.layout.on_resize(() => {
+      this.parent.request_layout(true)
+    })
+  }
+
+  get renderer(): GlyphRenderer<RadialGlyph> | null {
+    const {renderer} = this.model
+    if (renderer == "auto") {
+      const renderers = this.plot_view.model.renderers.filter((r): r is GlyphRenderer<RadialGlyph> => {
+        return r instanceof GlyphRenderer && r.glyph instanceof RadialGlyph
+      })
+      switch (renderers.length) {
+        case 0: {
+          logger.warn("can't find any radial glyph renderers")
+          return null
+        }
+        case 1: {
+          return renderers[0]
+        }
+        default: {
+          logger.warn("found multiple radial glyph renderers; choosing the first one")
+          return renderers[0]
+        }
+      }
+    } else {
+      return renderer
+    }
+  }
+
+  get glyph_view(): RadialGlyphView | null {
+    const {renderer} = this
+    if (renderer == null) {
+      return null
+    } else {
+      const rv = this.plot_view.views.get_one(renderer)
+      return rv.glyph_view as RadialGlyphView
+    }
+  }
+
+  protected _paint(_ctx: Context2d): void {
+    const {glyph_view} = this
+    if (glyph_view == null) {
+      return
+    }
+
+    const start = Math.ceil(uniforms.min(glyph_view.radius))
```

Comment:
```suggestion
    const start = Math.ceil(uniforms.min(glyph_view.radius)) || 1
```

So the first one is never 0.